question-servers/freeform.js

```diff
@@ -56,6 +57,13 @@ module.exports = {
             return '';
         } else if (phase == 'file') {
             return '';
+        } else if (phase == 'get_dependencies') {
```

Comment:
I was trying to match the convention of phases being verbs, but it doesn't matter either way to me.